### PR TITLE
Cleanup security scheduling

### PR DESCRIPTION
**Pull request by paolostivanin**
* merged security/alp/selinux and security/selinux
* merged security/alp/fips_crypt_core and security/fips_crypt_core
* removed enable_selinux
* refactor selinux_setup

- Related ticket: https://progress.opensuse.org/issues/137417
- Verification runs:
  - ALP: https://openqa.suse.de/tests/12551479
  - 15-SP5:
    - x86_64: https://openqa.suse.de/tests/12560219
    - s390x: https://openqa.suse.de/tests/12561681
    - aarch64: https://openqa.suse.de/tests/12561702
  - 12-SP5: https://openqa.suse.de/tests/12551472
  - Tumbleweed: https://openqa.opensuse.org/tests/3651396

### CODE REVIEW

**Comment by realcharmer** [comment]
LGTM

**Comment by realcharmer** [comment]
Looks good with the changes.

File: tests/security/selinux/selinux_setup.pm [hunk 1]
```diff
@@ -13,109 +12,63 @@ use testapi;
...
+        assert_script_run("tar -xzf $policy_archive");
+        assert_script_run("rpm -ivhU --nosignature --nodeps --noplugins ./selinux*$policy_date*.rpm", timeout => 120);
     }
+}
+
+sub run {
+    my ($self) = @_;
+
+    # on ALP selinux is enabled and set to enforcing by default
+    if (is_alp || is_sle_micro('>=6.0')) {
```
Comment (jknphy) [inline tests/security/selinux/selinux_setup.pm:56]: `is_sle_micro('>=6.0'))` will be alp so I don't think it is needed.

Comment (paolostivanin) [reply tests/security/selinux/selinux_setup.pm:56]: will it be ALP or will it be SLE Micro with alp codebase? 🤔  I honestly have no idea, but we can clean it up once we know more

**Comment by tjyrinki** [comment]
Very nice clean-up for future maintainability, thank you!

**Pull request merged**